<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="gfxBase">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>841</width>
      <height>681</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>277</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="lblWidth">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtWidth">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSep">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblStep">
       <property name="text">
        <string>h</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtXStep">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblStep_2">
       <property name="text">
        <string>tau</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtTStep">
       <property name="text">
        <string>0.01</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblNum">
       <property name="text">
        <string>Num</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtNum">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
	 <widget class="QLabel" name="label">
       <property name="text">
        <string>Explicit/Implicit</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSlider" name="sdrImplicitness">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnContinue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Continue</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Problem</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="rbtnProbHeat">
        <property name="text">
         <string>Simple heat conduction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnProbLoc">
        <property name="text">
         <string>Heat localization</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Scheme</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="rbtnSchemeLinear">
        <property name="text">
         <string>Linear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnSchemeNonLinear">
        <property name="text">
         <string>Non-linear</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnSchemeIter">
        <property name="text">
         <string>Iterating</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnReset</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtWidth</sender>
   <signal>editingFinished()</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnContinue</sender>
   <signal>released()</signal>
   <receiver>MainWindow</receiver>
   <slot>cont()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnProbHeat</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnProbLoc</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnSchemeIter</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnSchemeLinear</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbtnSchemeNonLinear</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>reset()</slot>
  <slot>cont()</slot>
 </slots>
</ui>
